AtTools/AtEDistortionModel.h

```diff
@@ -0,0 +1,90 @@
+#ifndef ATEDISTORTIONMODEL_H
+#define ATEDISTORTIONMODEL_H
+
+#include "AtSpaceChargeModel.h"
+
+#include <Rtypes.h> // for Double_t
+
+#include <fstream>
+#include <type_traits> // for add_pointer_t
+#include <unordered_map>
+
+class AtDigiPar;
+/**
+ * @brief Model for E-field distortion correction
+ *
+ *
+ *
+ *     The implemented correction is applied to every hit
+ *     Radial, transversal and longitudinal (z) corrections as a function of radial distance and z (map keys).
+ *     The Class EField mapping contains the correction values
+ */
```

Comment:
This doxygen comment will refer just to the struct EFieldCorr, which I don't think is the intention? It seems to be describing the entire model so should be above the actual model class.